gensrc/thrift/AgentService.thrift

```diff
@@ -116,6 +116,22 @@ struct TCreateTabletReq {
     20: optional bool create_schema_file = true;
 }
 
+struct TCreateTableReq {
+    1: optional list<TCreateTabletReq> create_tablet_reqs;
+    2: optional i32 timeout = 0; // unit: second
+}
+
+struct TAbortTxnReq {
+    1: required Types.TTransactionId txn_id;
```

Comment:
All fields in thrift/protobuf should be optional.

Reply:
Fixed